controller/contribution/revision.php

```diff
@@ -745,7 +751,20 @@ protected function process_steps()
 
 		$hash = $this->package->get_md5_hash();
 
-		$result = $this->run_step($step['function']);
+		$ignore_validation_errors = $this->request->variable('ignore_validation_errors', 0);
+
+		if ($ignore_validation_errors && $step['function'][1] == 'translation_validate')
```

Comment:
@VSE or @DavidIQ - I really dislike this line as it feels like such a dirty hack. Do you know if there's a more elegant way to check that the type of revision we are dealing with at this point is a translation?

Reply:
I don't know, but "elegant" is not a word I would ever associate with Titania's code 😉

Reply:
I think this is fine, but will need to make sure that submitting extensions and styles and other contributions aren't broken as I'm not sure what happens when that function key isn't present or isn't an array.